entities/build.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+
+# Absolute path to the directory containing this script
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
Because you go on to use many paths without the `$DIR` prefix, I say instead `cd "$( dirname "${BASH_SOURCE[0]}" )"` so that we can just assume that everything is relative to the script.